lnurl/explainer.md

```diff
@@ -0,0 +1,352 @@
+# LNURL
+
+LNURL is a standard for streamlining the process of enabling a client/consumer to conduct certain Lightning Network based operations with a web service that is connected to the Lightning Network. The client can be a wallet application, or even another web service. LNURL Flows automate some otherwise manual steps when executing these operations between two or more parties. This allows web service providers that make use of Lightning Network functionality to deploy API endpoints that can be interacted with autonomously by users implementing one or more of the various flows.
```

Comment:
```suggestion
LNURL allows web service providers to use Lightning Network functionality in their API endpoints that clients who have implemented one or more of the various flows can interact with autonomously. The client can be a wallet application or even another web service.

Several Flows are currently part of the LNURL specifications. They are; Pay, Withdraw, Channel, and Auth. These Flows automate some otherwise manual steps for certain Lightning Network-based operations and standardize the HTTP request and responses.
```